teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown')
+    super(age, name)
```

Comment:
- [ ] You are missing the `parent_permission` parameter for the `#initialize` and `#super` methods. Kindly add it.

Reply:
@TedLivist I have adjusted it. I felt it was unnecessary.